app/views/pregnancies/_patient_dashboard.html.erb

```diff
@@ -1,26 +1,26 @@
 <!-- pledge partial was here -->
 
-<div class="col-md-4" >
+<div class="col-sm-4" >
   <%= f.fields_for pregnancy.patient do |p| %>
     <%= p.text_field :name, label: 'First and Last Name', autocomplete: 'off' %>
     <%= p.text_field :primary_phone, label: 'Phone number', autocomplete: 'off' %>
   <% end %>
 </div>
-<div class="col-md-4">
+<div class="col-sm-4">
   <div class="row">
-    <div class="col-md-6">
+    <div class="col-sm-6">
       <%= f.select :last_menstrual_period_weeks, options_for_select(weeks_options, pregnancy.last_menstrual_period_weeks ), label: 'LMP at intake', autocomplete: 'off', help: "Currently: #{pregnancy.last_menstrual_period_display_short}" %>
     </div>
-    <div class="col-md-6 hide-label">
+    <div class="col-sm-6 hide-label">
       <%= f.select :last_menstrual_period_days, options_for_select(days_options, pregnancy.last_menstrual_period_days ), label: '.', autocomplete: 'off', help: "Called on: #{pregnancy.initial_call_date}" %>
     </div>
   </div>
   <div class="row">
-    <div class="col-md-12">
+    <div class="col-sm-12">
```

Comment:
Ah, I wasn't sure if it had been done yet. I saw them in the files I was working on so just made the changes there.